client/elife-theme/src/index.js

```diff
@@ -59,7 +59,5 @@ export default {
   transitionTimingFunction: 'ease',
   transitionDelay: '0',
 
-  cssOverrides: {
-    Button,
-  },
+  cssOverrides,
```

Comment:
This is in conflict to what I did in https://github.com/elifesciences/elife-xpub/pull/248

Reply:
I think this would move faster if we agree on something and then don't change things that are not related to the issue in commits?

Reply:
OK. Can we agree on this?